src/Tickets/Seating/Orders/Controller.php

```diff
@@ -531,9 +528,11 @@ public function get_localized_data( ?int $post_id = null ): array {
 	private function register_assets(): void {
 		Asset::add(
 			'tec-tickets-seating-admin-seats-report',
-			$this->built_asset_url( 'admin/seatsReport.js' ),
+			'admin/seatsReport.js',
 			Tickets_Main::VERSION
 		)
+			->add_to_group_path( 'tec-seating' )
+			->prefix_asset_directory( false )
```

Comment:
Having to call a second method, `prefix_asset_directory`, on each registration looks tiring and error-prone.
My understanding is the group, `tec-seating`, and whether the `prefix_asset_directory` should be true or not are coupled, why not make the `prefix_asset_directory` a **group** property, not an asset property?

Reply:
Completely agree - Wanted to fix it when i discovered that was needed on Friday but there wasnt enough time. Will open a PR today amending that.